docker/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the code.
-FROM public.ecr.aws/unocha/unified-builder:8.0-stable as builder
+FROM public.ecr.aws/unocha/unified-builder:8.1-stable as builder
```

Comment:
Do we want 8.1 for the builder if we're going to 8.2?

Reply:
We do not. But ther eis no 8.2 b8uilder and I'm not about to bake one just for GMS. So this terrible solution is what I went with.

As a follow-up, the GMS theme should be updated so it doesn't need sassing.

Reply:
Just dropped the sass build step and the 8.1 builder altogether :-)